python/cudaq/__init__.py

```diff
@@ -107,6 +107,7 @@ def disable_jit():
 # Expose chemistry domain functions
 from .domains import chemistry
 from .kernels import uccsd
+import lib
```

Comment:
Importing as `from .lib import *` got all the ctests to work (including ` qpp_H2MoleculeTester.checkHamiltonian_Sampling`)